go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/gruntwork-io/gruntwork-cli v0.5.1
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/jstemmer/go-junit-report v0.9.1
+	github.com/magiconair/properties v1.8.0
```

Comment:
Is this part of this PR?

Reply:
Yea. I ran `go mod tidy` and it showed up, indicating that we forgot to add this in in some other PR.